src/Card.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace PodPoint\Payments;
+
+class Card
+{
+    /**
+     * The uid or unique identifier for a payment method.
+     *
+     * @var string
+     */
+    public $uid;
+
+    /**
+     * Last 4 digits of the card.
+     *
+     * @var string
+     */
+    public $last4;
+
+    /**
+     * Brand of the card.
+     *
+     * @var string
+     */
+    public $brand;
+
+    /**
+     * Funding of the card.
```

Comment:
what is this?

Reply:
Card funding type. Can be credit, debit, prepaid, or unknown.

We are using that field in the cardTransformer so I kept it: 
See: https://github.com/Pod-Point/api3/blob/9a7da51c0976519df13906bc17c8ba006acad78a/app/Api/Transformers/CardTransformer.php